build/ci-deploy.yml

```diff
@@ -29,7 +29,7 @@ stages:
       displayName: Provision Resource Group
       inputs:
         azureSubscription: $(ConnectedServiceName)
-        azurePowerShellVersion: latestVersion
+        azurePowerShellVersion: '6.3.0'
```

Comment:
can this be a variable? Seems like it should be consistent

Reply:
I'm going to revert this change since it didn't solve the issue.